examples/v1/workflows/dag.go

```diff
@@ -0,0 +1,65 @@
+package v1_workflows
+
+import (
+	"fmt"
+
+	v1 "github.com/hatchet-dev/hatchet/pkg/v1"
+	"github.com/hatchet-dev/hatchet/pkg/v1/task"
+	"github.com/hatchet-dev/hatchet/pkg/v1/workflow"
+	"github.com/hatchet-dev/hatchet/pkg/worker"
+)
+
+type SimpleOutput struct {
+	Step int
+}
+
+type DagResult struct {
+	Step1 SimpleOutput
+	Step2 SimpleOutput
+}
+
+func DagWorkflow(hatchet *v1.HatchetClient) workflow.WorkflowDeclaration[any, DagResult] {
+
+	simple := v1.WorkflowFactory[any, DagResult](
+		workflow.CreateOpts[any]{
+			Name: "simple-dag",
+		},
+		hatchet,
+	)
+
+	step1 := simple.Task(
+		task.CreateOpts[any]{
```

Comment:
same here I think - ideally we'd not need to provide this? idk if it's possible 🤷